published/012-2016-August-24.md

```diff
@@ -0,0 +1,130 @@
+
+> Beaker is a browser, but it's specifically a solution for putting decentralization protocols and APIs together, so we can stay lean and work together. It's called "Beaker" to signify that it's a place for experimentation, and a place to take risks. It's a pre-standards browser, so to speak; a place to try a technology without writing a spec first. And, it's been made to pursue that mandate of decentralization: that we disconnect presentation from services, that we improve confidentiality, and that we remove the advantages that keep large businesses in control of the Web.
+
+You can find out more about it [here](https://pfrazee.github.io/2016/08/10/beaker-browser-0.1.html), [watch the demo video](https://www.youtube.com/watch?v=nKHJ4rLN9mo), read the [Hacker News discussion](https://news.ycombinator.com/item?id=12270762), or go straight to the GitHub repo and download the binary [here](https://github.com/pfrazee/beaker). Some future plans involve possibly integrating [scuttlebutt](https://scuttlebot.io/more/protocols/secure-scuttlebutt.html).
+
+### Primoca
+
+[Primoca](https://www.primoca.com/), an interactive code sharing environment, has hooked up a browser extension to a local IPFS daemon, which means that publishing code on Primoca is dependant entirely on IPFS. You can test it out [here](https://run.primoca.com/QmWb4sq11uZEdHk78nSocFVNRSMFcW3NnUrvH3ZnFvbgyT#script=QmQXp7maEfSs4w1QtpJ1VDpRvStNmXK4tXPKP7VtjVy63C). 
+
+### Mango: Git completely Decentralized
+
+@alexberegszaszi wrote an [outstanding Medium article](https://medium.com/@alexberegszaszi/mango-git-completely-decentralised-7aef8bcbcfe6#.7i4vgwihz) introducing [Mango](https://github.com/axic/mango), a backend for Git that uses Ethereum and IPFS (or Swarm). In their system, all Git objects (metadata and data) are stored on IPFS, while an Ethereum smart contract provides means for access control and stores the pointers to the latest repository revisions.
+
+### How to serve a website over IPFS
+
+Etienne Laurin wrote up a guide on [how to serve a website on IPFS](http://www.atnnn.com/p/ipfs-hosting/), which lays out the workflow for going from a traditional website to an IPFS-based one. The site itself, of course, is also hosted on IPFS. If you're looking to publish your own site, this is a good place to start.
+
+### Swag
+
+We have now emailed out stickers to a ton of hackerspaces and individuals. Let us know if you would like some: @RichardLitt would be happy to send out a few more.
+
+## Contributors
+
+Across the entire IPFS GitHub organization, the following people have committed code, created issues, or made a comment on GitHub between __Date__ (noon, GMT) and __Date__. We're autogenerating this list using [this tool](//github.com/ipfs/weekly/blob/master/tools/get_commits.py) and [this other tool](//github.com/richardlitt/name-your-contributors), so please let us know if your name isn't here.
+
+- __Contributors__
+
+TODO
+- [ ] Grab the newest contributors for this month
+
```

Comment:
Not sure if the contributors tools grabs orbit repos, but @claus should be on the list.